[2018-Oct-12 11:59:58][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 11:59:58][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 103, in _run
    output_placement_list.append((outputs_dict[str(replicate_counter)+row+":"+column],(output_container_list[0],plate_layout[well_counter])))
IndexError: list index out of range

[2018-Oct-12 12:00:32][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:00:32][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    output_placement_list.append((outputs_dict[str(replicate_counter)+row+":"+column],(output_container_list[0],plate_layout[well_counter])))
IndexError: list index out of range

[2018-Oct-12 12:02:05][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:02:05][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    output_placement_list.append((outputs_dict[str(replicate_counter)+row+":"+column],(output_container_list[0],plate_layout[well_counter])))
IndexError: list index out of range

[2018-Oct-12 12:02:31][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:02:31][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:03:10][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:03:10][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:03:53][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:03:53][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:49:30][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:49:30][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 108, in _run
    if plate_layout[well_counter] == "P:5":
IndexError: list index out of range

[2018-Oct-12 12:50:29][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:50:29][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:52:03][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:52:03][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:56:09][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:56:09][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:58:28][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:58:28][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 12:59:53][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 12:59:53][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 13:00:41][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 13:00:41][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

[2018-Oct-12 13:02:25][AutoplacementQPCR384][CRITICAL] Encountered a IndexError exception: list index out of range
[2018-Oct-12 13:02:25][AutoplacementQPCR384][ERROR] Stack trace below:
Traceback (most recent call last):
  File "C:\Users\dforth\PycharmProjects\untitled\venv\lib\site-packages\clarity_scripts-0.9.dev0-py3.6.egg\EPPs\common.py", line 126, in run
    self._run()
  File "C:\Users\dforth\PycharmProjects\clarity_scripts\scripts\autoplacement_qpcr_384.py", line 104, in _run
    print(plate_layout[well_counter])
IndexError: list index out of range

